### ISSUE TITLE
Proposed RFC Suggestion: Renaming sig-presentation

### ISSUE THREAD

**Issue by wintermute-motherbrain** [o3de/sig-presentation#4]
### Summary:
Some folks have expressed that the name "sig-presentation" does not obviously indicate that graphics and audio fall under the SIG.  This RFC is to solicit feedback on whether or not sig-presentation should be renamed and if so, what should the new name be.

### What is the motivation for this suggestion?
Why is this important?  

We want graphics questions to flow through sig-presentation and not to other sigs.  This requires discoverability to be straightforward for new folks to O3DE.

What should be the outcome be if this suggestion is implemented?

We either keep the current name or a new name is chosen.

###  Next Steps

Provide comments to this RFC indicating if you feel the name should be changed and if so, propose a new name.  We will then bring this to the next monthly meeting agenda.

**Comment by ppinfel** [comment]
suggestion for name: sig-rendering-audio

**Comment by antonmic** [comment]
sig-graphics (sig-graphics-audio could work too, but I have yet to hear a convincing argument for putting these two together, audio talk will be spam for graphics people and graphics talk will be spam for audio people, it's entirely unproductive)
Alternatively sig-rendering / sig-rendering-audio would also be good

**Comment by wintermute-motherbrain** [comment]
I think the issue with audio is its too small to be its own sig with its own charter, etc and thus it needs to fall in with another sig.

**Comment by moudgils** [comment]
Is renaming to sig-graphics and moving audio to sig-core an option? I would vote for that.

**Comment by OBWANDO** [comment]
Sig-core has enough to deal with. 

I'm also with the idea of sig-graphics-audio

The thing about having audio with graphics is that some of the trends are pushing towards having GPUs doing the audio side. The only other thing would be for another SIG to accept the audio part, which could be sig-content, but thats iffy.
Having another dedicated sig (like sig-audio) with no one in it does not make much sense.

**Comment by wintermute-motherbrain** [comment]
@OBWANDO Today we had our latest sig-presentation [meeting](https://github.com/o3de/sig-presentation/issues/5) where we discussed this RFC and put it to a vote.  The sig has requested to change the name from sig-presentation to sig-graphics-audio.  Can we start the process to renaming both the sig-presentation repo (this repo) as well as updating the discord channel names?  Free to chat further on discord as well.

**Comment by OBWANDO** [comment]
Yes indeed, I'll begin the process of renaming. Great to see the SIG bring it to a vote and take action!
